src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -105,6 +105,14 @@ uiModules.get('apps/management')
               name: bucket.key
             };
           }), 'name');
+        })
+        .catch(err => {
+          if (err.body && err.body.error && err.body.error.caused_by.type === 'index_not_found_exception') {
```

Comment:
Lets just use `get()` here, or also check that `err.body.error.caused_by` is truthy before trying to access its `.type` property.

Reply:
I think it would be a lot easier to read if it were just 

```js
const type = _.get(err, 'body.error.caused_by.type');

if (type === 'index_not_found_exception') {
```